### PR TITLE
Improve rakefile/readme

### PR DESCRIPTION
**Pull request by McIntireEvan**
* Add "import:semester" task to import flat file course data
* Add "scrape:semester" task to scrape a specific semester
* Add a TODO for an export command
* Add a TODO to improve test_scrape
* Update README to properly instruct on cloning umdio-data for testing
* Add `import:map`, update script to allow flexibility

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ To work on umd.io, or to run your own instance, start by forking and cloning thi
    - You might need to run docker-related commands with `sudo` if you're a linux user
 4. Run the scrapers `./umdio.sh scrape`
    - You might need to `chmod +x umdio.sh`
+5. Clone the umdio-data submodule
+   - `cd data && git submodule init && git submodule update`
```
Comment (tybug) [inline README.md:31]: just `git submodule update --init` is probably cleaner?

**Review by tybug — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (McIntireEvan) [reply README.md:31]: Nice catch - goes to show my git submodules knowledge, haha

Comment (tybug) [reply README.md]: oh you don't even need the `cd data`, should have clarified that earlier

File: app/scrapers/map_scraper.rb [hunk 2]
```diff
@@ -10,16 +10,21 @@
...
+def write_map_array(data)
+  data.each do |e|
+    $DB[:buildings].insert_ignore.insert(:name => e[:name], :code => e[:code], :id => e[:number].upcase, :long => e[:lng], :lat => e[:lat])
+    logger.info($prog_name) {"inserted #{e[:name]}"}
+  end
 end
+
+$DB[:buildings].delete
+
+uri = ARGF == 1 ? ARGV[0] : url
```
Comment (tybug) [inline app/scrapers/map_scraper.rb:28]: uri is never used?

Comment (McIntireEvan) [reply app/scrapers/map_scraper.rb:28]: Argh, typo. one moment

File: app/scrapers/map_scraper.rb [hunk 3]
```diff
@@ -10,16 +10,21 @@
...
+array = eval URI.open(url).read
+write_map_array(array)
```
Comment (tybug) [inline app/scrapers/map_scraper.rb:30]: should enable adding trailing newlines by default in whatever editor you use (vscode right? I kinda thought they had that turned on by default tbh)

Comment (McIntireEvan) [reply app/scrapers/map_scraper.rb:30]: Enabled!

**Review by DonIsaac — APPROVED** [review]

**Pull request merged**